DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: linelist
 Title: Tools to Import and Tidy Case Linelist Data
-Version: 0.0.36.9000
+Version: 0.0.38.9000
```

Comment:
not new, but are we using the last digit for versioning? So far our guidelines mention semantic major.minor.match

Reply:
The 9000 is an indicator for the user that things are still under development. It still works with semantic declarations in other packages, though (e.g. linelist (>= 0.0.38))